content/desktop/install/msi/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Deploy with Microsoft installer
+description: If you're an IT admin, learn how to deploy Docker Desktop with Microsoft installer
+keywords: msi, docker desktop, windows, installation
+---
+ 
+Docker offers a 64-bit Microsoft Installer (MSI) package designed to assist IT administrators in customizing and deploying Docker Desktop across multiple users or machines in an enterprise environment. 
+
+The MSI package supports various MDM (Mobile Device Management) solutions, making it ideal for bulk installations and eliminating the need for manual setups by individual users. With this package, IT administrators can ensure standardized, policy-driven installations of Docker Desktop, enhancing efficiency and software management across their organizations. Instead, it is installed and automatically launched the first time a developer signs in to their machine, simplifying the setup process.
```

Comment:
I just noticed, the last line here is not correct.

```
Instead, it is installed and automatically launched the first time a developer signs in to their machine, simplifying the setup process.
```

It is installed automatically, but we will not launch the app when the developer signs in.

Maybe we could change it to something like:

```
Instead, it is installed and available for developers when they sign in.
```